### PR TITLE
v1.3.5

### PR DESCRIPTION
**Pull request by MBueschelberger**
* Give opportunity to make slugs individual even if derived from the same kitem name

### CODE REVIEW

File: dsms/core/configuration.py [hunk 1]
```diff
@@ -58,6 +58,13 @@ class Configuration(BaseSettings):
         description="Repository of the triplestore for KItems in the DSMS",
     )
 
+    individual_slugs: bool = Field(
```
Comment (yoavnash) [inline dsms/core/configuration.py:61]: Perhaps `automatic_slugs` or `autocomplete_slug` would be a more self-explantory name.

File: dsms/knowledge/kitem.py [hunk 2]
```diff
@@ -362,18 +362,19 @@ def validate_ktype(cls, value: KType, info: ValidationInfo) -> KType:
...
-        slugified = _slugify(value)
-        if len(slugified) < 4:
-            raise ValueError("Slug length must have a minimum length of 4.")
-        if value != slugified:
-            raise ValueError(
-                f"`{value}` is not a valid slug. A valid variation would be `{slugified}`"
-            )
+            if Context.dsms.config.individual_slugs:
+                value += f"-{str(kitem_id).split('-', maxsplit=1)[0]}"
+            if len(value) < 4:
```
Comment (yoavnash) [inline dsms/knowledge/kitem.py:374]: This will allow names such as `a-12345678`

**Review by yoavnash — COMMENTED** [review]
Added two comments

**Comment by MBueschelberger** [comment]
This is an depricated PR due to previous developments. Hence I will close this PR.

**Pull request closed without merging**